honeybee_grasshopper_core/src/HB Run Recipe.py

```diff
@@ -107,24 +107,27 @@ def sort_results_by_grid(results, grid_names):
     # write the inputs JSON for the recipe
     inputs_json = _recipe.write_inputs_json(_folder_)
 
-    # execute the queenbee luigi CLI to obtain the results via CPython
-    queenbee_exe = os.path.join(hb_folders.python_scripts_path, 'queenbee.exe') \
-        if os.name == 'nt' else os.path.join(hb_folders.python_scripts_path, 'queenbee')
-    cmds = [queenbee_exe, 'luigi', 'translate', _recipe.path, _folder_,
-            '-i', inputs_json, '--workers', _cpu_count_]
-    if rad_folders.radlib_path:  # set the RAYPATH environment variable
-        cmds.extend(['--env', 'RAYPATH={}'.format(rad_folders.radlib_path)])
-    if rad_folders.radbin_path:  # set the PATH environment variable
-        cmds.extend(['--env', 'PATH={}'.format(rad_folders.radbin_path)])
-    cmds.append('--run')
+    genv = os.environ.copy()
+    genv['PATH'] = os.pathsep.join((rad_folders.radbin_path, genv['PATH']))
+    genv['RAYPATH'] = os.pathsep.join((rad_folders.radlib_path, genv['RAYPATH']))
+
+    # create command
+    command = '"{hb_recipe}" run {recipe_name} ' \
+        '"{project_folder}" -i "{user_inputs}" --workers {cpu_count} ' \
+        '--name {simulation_name}'.format(
+            hb_recipe=os.path.join(hb_folders.python_scripts_path, 'honeybee-recipe'),
+            recipe_name=_recipe.name, project_folder=_folder_,
+            user_inputs=inputs_json, cpu_count=_cpu_count_,
+            simulation_name=_recipe.simulation_id
+        )
```

Comment:
I see you're going the route of using a single sting instead of a list.  I guess this was the only way that you could get things to work with spaces in the file paths, is that right?

Reply:
Yes. This was the most reliable option that I found to handle commands on different OS regardless of white spaces.